Code/Framework/AzFramework/AzFramework/Physics/PhysicsScene.cpp

```diff
@@ -51,7 +51,10 @@ namespace AzPhysics
                 ->Attribute(AZ::Script::Attributes::Category, "Physics")
                 ->Method("GetOnGravityChangeEvent", getOnGravityChange)
                     ->Attribute(AZ::Script::Attributes::AzEventDescription, gravityChangedEventDescription)
-                ->Method("QueryScene", &Scene::QueryScene)
+                ->Method("QueryScene", [](Scene* selfPtr, const SceneQueryRequest* request)
```

Comment:
uber nit: Could just use the variable `self` and omit `Ptr` from the name (less Hungarian style notation)